### ISSUE TITLE
V10 planned?

### ISSUE THREAD

**Issue by bucheggerOnline** [fabarea/media#207]
Hi. I'm wondering if there will be soon an update to use your ext on Typo3v10?

**Comment by fabarea** [comment]
Yes, it will be (but no date to communicate) I have started the work for Vidi already here https://github.com/fabarea/vidi/tree/typo3-v10 (actually it is testable)

**Comment by Epoche-Napoleon** [comment]
Hello, 

is there already a timetable in the meantime? Is it possible to influence it through sponsorship?

Happy new year 2021!
Epoche Napoleon

**Comment by abvdveen** [comment]
Hi Fabian, are you planning on working on this in the near future?

**Comment by fabarea** [comment]
@abvdveen Yes, I am planning to take some time for that mid-May. 🤞 
EXT:vidi has already a branch compatible v10

**Comment by abvdveen** [comment]
@fabarea Thanks for letting me know. Hope you'll find some time for this 🤞 😊

**Comment by oberritter** [comment]
@fabarea Hi, there are still some errors in v10.  Do you think it will be finished until end of the year?

**Comment by fabarea** [comment]
@oberritter what errors?

**Comment by oberritter** [comment]
![fab_media_error_1](https://user-images.githubusercontent.com/15647992/132862983-20b043d7-4578-4a55-933c-8e08993c8a8e.png)

Line 98 media/Classes/View/Warning/ConfigurationWarning.php

This could fix the error:
```
 $connection = GeneralUtility::makeInstance(ConnectionPool::class)
            ->getConnectionForTable($tableName);
```

<img width="625" alt="fab_media_error_2" src="https://user-images.githubusercontent.com/15647992/132862987-9db195d0-0e59-4113-99d2-e48da6481176.png">

Line 32 media/Classes/Override/Backend/Form/FormResultCompiler.php

https://docs.typo3.org/c/typo3/cms-core/master/en-us/Changelog/9.3/Deprecation-84993-DeprecateSomeTSconfigRelatedMethods.html

**Comment by abvdveen** [comment]
@fabarea I recommend looking at the results of 'Scan Extension Files' under Upgrade in the BE. There you'll find the above mentioned deprecated method, among others, some of which have to be fixed.
